core/src/main/java/de/danoeh/antennapod/core/storage/DBTasks.java

```diff
@@ -341,6 +347,12 @@ public static synchronized Feed updateFeed(Context context, Feed newFeed, boolea
             e.printStackTrace();
         }
 
+        // We need to add to queue after items are saved to database
+        DBWriter.addQueueItem(
+                context,
+                itemsToAddToQueue.toArray(new FeedItem[itemsToAddToQueue.size()])
+        );
```

Comment:
Please only add line breaks if it otherwise makes the line too long. We don't usually write one parameter per line because that takes a lot of vertical space and fills it with white space.

Reply:
This line is too long. I changed it to be only partially split, but this reads way worse for me.

Reply:
You don't need the actual size here, just 0 is enough. (I know that's a really weird part of the Java API :D)

Reply:
0 is actually (slightly) less efficient, because it will cause Java to recreate a new array to fit this in, discarding the array that we just passed.

Reply:
Some benchmarks seem to say that the version without size is faster, especially on newer Java versions: https://www.baeldung.com/java-collection-toarray-methods

Anyway, I think it is more important to have code that is easier to read. I mean, the full array needs to be allocated anyways, and this is not on a critical code path. Using `[0]` will make it possible to reduce the number of lines from 4 to 1, making the code easier to skim

Reply:
Huh, TIL. Changed it to [0].